Jenkinsfile

```diff
@@ -12,7 +12,9 @@ pipeline {
           steps {
             container('jetty-build') {
               timeout( time: 120, unit: 'MINUTES' ) {
-                mavenBuild( "jdk17", "clean install", "maven3")
+                // configuration needed for errorprone but not for normal build
+                sh "mv .mvn/.jvm.config .mvn/jvm.config"
```

Comment:
Is there any harm in leaving this configuration enabled for both builds?

Reply:
nope no issue with jdk11.
maybe to ease this for every build even local? (i.e just having a file ` .mvn/jvm.config`)
wdyt?

Reply:
Yes, lets leave it enabled for any JVM. (even JDK 18 and JDK 19)

Reply:
fixed

Reply:
Going to wait for a green build now ... :-)